db/migrate/20210701184921_add_location_to_users.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddLocationToUsers < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :location, :jsonb
```

Comment:
This migration seems useless. If you don't read value of `location` then it can be removed.